integration-tests/src/test/resources/loggingexpdomainonpvwlst.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
```

Comment:
is this a new file? remove "2018" from copyright notice

Reply:
"2018" is removed